source/extensions/common/wasm/wasm_vm.cc

```diff
@@ -19,20 +19,19 @@ namespace Common {
 namespace Wasm {
 
 proxy_wasm::LogLevel EnvoyWasmVmIntegration::getLogLevel() {
-  switch (ENVOY_LOGGER().level()) {
-  case spdlog::level::trace:
-    return proxy_wasm::LogLevel::trace;
-  case spdlog::level::debug:
-    return proxy_wasm::LogLevel::debug;
-  case spdlog::level::info:
-    return proxy_wasm::LogLevel::info;
-  case spdlog::level::warn:
-    return proxy_wasm::LogLevel::warn;
-  case spdlog::level::err:
-    return proxy_wasm::LogLevel::error;
-  default:
-    return proxy_wasm::LogLevel::critical;
+  static const absl::flat_hash_map<spdlog::level::level_enum, proxy_wasm::LogLevel> log_level_map =
```

Comment:
Ditto, Not fan of this kind of style change to improve coverage report artificially.

No static const of non-POD, or use pointer (`CONSTRUCT_ON_FIRST_USE`) macro.

Reply:
reverted as well.